graviti/dataframe/indexing.py

```diff
@@ -7,56 +7,116 @@
 
 from typing import TYPE_CHECKING, Any, Generic, Iterable, TypeVar, Union, overload
 
+from graviti.utility import NestedDict
+
 if TYPE_CHECKING:
+    from graviti.dataframe.column.series import Series as ColumnSeries
     from graviti.dataframe.dataframe import DataFrame
     from graviti.dataframe.row.series import Series as RowSeries
-    from graviti.dataframe.series import SeriesBase
+
+    _S = TypeVar("_S", RowSeries, ColumnSeries)
 
 _T = TypeVar("_T", int, str)
 
 
-class SeriesILocIndexer(Generic[_T]):
-    """Index class for Series.iloc."""
+class SeriesILocIndexerBase(Generic["_S"]):
```

Comment:
since the SeriesBase is been removed, do we still need IndexerBase?

Reply:
`RowSeriesILocIndexer` and `ColumnSeriesILocIndexer` have the same function `__getitem__`

Reply:
Are they really the same?
Can you provide some example code?

Reply:
Remove the `IndexerBase` now.